src/api/app/models/bs_request_action.rb

```diff
@@ -287,8 +287,14 @@ def notify_params(ret = {})
   end
 
   def contains_change?
-    return sourcediff.present?
-  rescue BsRequestAction::DiffError
+    source_info_target = Backend::Api::Sources::Package.source_info(target_project, target_package, nofilename: 1)
+    source_info_source = Backend::Api::Sources::Package.source_info(source_project, source_package, nofilename: 1)
```

Comment:
[nitpick] Would change to ``source_info_source_package`` and ``source_info_target_package`` because ``source_info_source`` sounded really weird to me and I needed to read the whole line to understand what it means ...

Reply:
I also don't like the name. But I also didn't want to make it even longer^^ I'll change it